.github/workflows/main.yml

```diff
@@ -61,10 +58,13 @@ jobs:
         uses: ./.github/actions/ci
 
       - name: Building Visual Regression Tests application for UI components
-        run: yarn visual-testing-app:build
+        run: pnpm visual-testing-app:build
+
+      - name: Installing Chrome via Puppeteer
+        run: node ./node_modules/puppeteer/install.js
```

Comment:
ℹ️ We're explicitly installing Chrome via Puppeteer as at times running the tests will fail due to Chrome / Chromium not being found.
This workaround is based on the suggestion from this issue: https://github.com/puppeteer/puppeteer/issues/9533.